qaframework-bdd-tests/src/test/resources/features/refapp-2.x/provider/provider.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Provider Management
+
+  Background: 
+    Given User clicks the system administration link on the home page
+    When User clicks the manage Provider link on the administration page
+
+  @selenium
+  @provider
+  Scenario: Adding Provider
+    And User clicks on add provider link
+    And User fills the provider form
+    And User searches for the created provider
+    Then User deletes provider forever
+
+  @selenium
+  @provider
+  Scenario: Editing Provider
+    And User searches for the created provider
+    Then User edits provider details
```

Comment:
which provider is the testing  editing in this scenario?

Reply:
In the `Scenario: Editing Provider`, which provider is being edited? In other words where is the provider created so that it can be edited?

Reply:
So ideally the tests only edits using identifier with given uui depending on the shared editProvider Test

Reply:
Searches for the identifier and the results return a provider attached to that identifier which is therefore edited